io/eolearn/io/processing_api.py

```diff
@@ -6,84 +6,51 @@
 import numpy as np
 
 from sentinelhub import WebFeatureService, MimeType, SentinelHubDownloadClient, DownloadRequest, SHConfig,\
-    bbox_to_dimensions
+    bbox_to_dimensions, parse_time_interval, DataSource
 import sentinelhub.sentinelhub_request as shr
 
 from eolearn.core import EOPatch, EOTask, FeatureType
 
 LOGGER = logging.getLogger(__name__)
 
 
-class SentinelHubInputTask(EOTask):
-    """ A processing API input task that loads 16bit integer data and converts it to a 32bit float feature.
-    """
-    def __init__(self, data_source, size=None, resolution=None, bands_feature=None, bands=None, additional_data=None,
-                 maxcc=1.0, time_difference=None, cache_folder=None, max_threads=None, config=None,
-                 bands_dtype=np.float32):
+class SentinelHubInputBase(EOTask):
+    ''' Base class for Processing API input tasks
```

Comment:
Could you replace `'''` with `"""` everywhere. Using double-quotes for docs string is the standard way.

Reply:
I keep repeating this mistake...